productivity-suite/app/fragments/todos/src/App.tsx

```diff
@@ -32,6 +38,16 @@ const App: React.FC<{
 
   if (!currentUser || !listName) return <></>;
 
+  function setTodos(todos: Todo[], dispatch = true) {
+    setTodosState(todos);
```

Comment:
`setTodos` and `setTodosState`.... we may need better names 😓

Reply:
Does React State work a bit different to Qwik Store?
If we bundle these together will it trigger more rendering changes that we want?

Reply:
no, the issue is that actually `setTodos` simply calls `setTodosState` but also dispatches the event, actually I think we can just keep the original `setTodos` (instead of renaming it to `setTodosState`) and call the other function `setTodosAndDispatchUpdate` or something along those lines?

Reply:
I've refactored this bit so now there's only `setTodos` 🙂👍